src/lib/util.ts

```diff
@@ -0,0 +1,14 @@
+import { statPromise } from './misc';
```

Comment:
[IMO] I suppose `log` and `statFile` function can place in misc.ts. "util" and "misc" have the same responsibility so we may confuse which file should use.

Reply:
Seems legit. I'll dissolve these utils into misc. yet `misc` file looks a little too bloated, I'd reorganize them with multiple modules later.

Reply:
`misc` renamed to `server` which only embraces server related function. other __real misc__ functions have been moved to `util`.